tests/PHPStan/Command/ErrorFormatter/BaselineNeonErrorFormatterTest.php

```diff
@@ -207,6 +207,9 @@ public function outputOrderingProvider(): \Generator
 			new Error('Second error in a different file', 'TestfileB', 4),
 			new Error('Error #1 in a different file', 'TestfileB', 5),
 			new Error('Second error in a different file', 'TestfileB', 6),
+			new Error('Error with Windows path separators', 'TestFiles\\TestA', 1),
```

Comment:
It should be named a "dir separator", not a "path separator" as the later means something else, see https://www.php.net/manual/en/dir.constants.php